### DISCUSSION TITLE
information box for geopandas plot

### DISCUSSION THREAD

**Discussion by rh-afk** [opengeos/leafmap#546] [Q&A] [answered]
Hi All,

how to customize information box for geopandas plot for example become pop up (same with marker information when we click the marker and will be a pop up there) so its not stay forever in map for that information box, incase we have many gpf plot , the map will be full with that box and the second question how to on/off this box information.

![image](https://github.com/opengeos/leafmap/assets/141318843/4120e739-46e8-4647-a52d-9b9d07a64c70)
thanks

**Comment by giswqs** [comment, marked as answer]
You can click the icon to hide the box, or set `info_mode=None` to turn it off
